extra_data/cli/make_virtual_cxi.py

```diff
@@ -106,11 +97,17 @@ def main(argv=None):
         sys.exit("ERROR: Don't have write access to {}".format(out_dir))
 
     log.info("Reading run directory %s", run_dir)
-    run = RunDirectory(run_dir, inc_suspect_trains=(not args.exc_suspect_trains))
-    det = _get_detector(run, args.min_modules)
-    if det is None:
-        sys.exit(f"No {_detectors()} sources found in {run_dir}")
+    inc_suspect = not args.exc_suspect_trains
+    run = RunDirectory(run_dir, inc_suspect_trains=inc_suspect)
+
+    _, det_class = identify_multimod_detectors(run, single=True)
+
+    min_modules = args.min_modules
+    if min_modules is None:
+        det_n_modules = getattr(det_class, 'n_modules', 0)
+        min_modules = (det_n_modules // 2) + 1
```

Comment:
As we're documenting that this is half + 1, we should probably respect `--n-modules` for JUNGFRAU.